widget_sample_gauge_widget/package.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "dashboard_widget_plugin_template",
+    "private": true,
+    "version": "0.0.0",
+    "type": "module",
+    "scripts": {
+        "dev": "vite",
+        "build": "tsc && vite build",
+        "// Support dev server remote entry file": "https://github.com/originjs/vite-plugin-federation/issues/525",
+        "build:watch": "vite build --watch & vite preview",
+        "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+        "preview": "vite preview"
+    },
+    "dependencies": {
+        "@mui/material": "^5.16.7",
+        "@mui/styled-engine-sc": "^5.14.12",
+        "react": "^18.2.0",
+        "react-dom": "^18.2.0",
+        "styled-components": "^5.3.11"
+    },
+    "devDependencies": {
+        "@amcharts/amcharts5": "^5.10.11",
+        "@amcharts/amcharts5-fonts": "^5.0.1",
+        "@moderepo/bizstack-console-sdk": "file:../../bizstack-console/app/moderepo-bizstack-console-sdk-1.8.0.tgz",
```

Comment:
NOTE: I am installing the BizConsole SDK from local directory because the SDK version 1.8 has not been released yet. I will not merge this PR until we release the 1.8 SDK and I change this to installing from remote.